src/hooks/useCroct.test.tsx

```diff
@@ -1,28 +1,31 @@
-import {renderHook} from '@testing-library/react-hooks';
+import {renderHook} from '@testing-library/react';
 import croct, {Plug} from '@croct/plug';
 import {useCroct} from './useCroct';
 import {CroctContext, CroctProviderProps} from '../CroctProvider';
 
 describe('useCroct', () => {
     it('should fail if used out of the <CroctProvider/> component', () => {
-        jest.spyOn(console, 'error').mockImplementation();
+        jest.spyOn(console, 'error')
+            .mockImplementation();
 
-        const {result} = renderHook<CroctProviderProps, Plug>(() => useCroct());
+        expect(() => {
+            renderHook<Plug, CroctProviderProps>(() => useCroct());
+        })
+            .toThrow('useCroct() can only be used in the context of a <CroctProvider> component.');
```

Comment:
Fixing the import, we can do it like this, since it

```suggestion
        const {result} = renderHook(() => useCroct());

        expect(result.error)
            .toEqual(new Error('useCroct() can only be used in the context of a <CroctProvider> component.'));
```